src/Components/Admins/table.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Row from './Row';
+import styles from './table.module.css';
+
+const Table = ({ admins, deleteAdmin }) => {
+  return (
+    <table className={styles.table}>
+      <thead>
+        <tr>
+          <th id="name">Name</th>
+          <th id="last-name">Last Name</th>
+          <th id="email">Email</th>
+          <th id="actions">Actions</th>
```

Comment:
```suggestion
          <th>Name</th>
          <th>Last Name</th>
          <th>Email</th>
          <th>Actions</th>
```
Those id properties are not used